demo/genome_Snakefile.smk

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env snakemake
+
+'''
+It will only run through the workflow(eg. genome) sub directory.
+'''
+##### SETUP #####
+
+import oncopipe as op
+
+SAMPLES = op.load_samples("data/samples.tsv")
+GENOME = op.filter_samples(SAMPLES, seq_type = "genome")
+
+
+##### REFERENCE_FILES WORKFLOW #####
+
+
+subworkflow reference_files:
+    workdir:
+        "reference/"
+    snakefile:
+        "../workflows/reference_files/2.4/reference_files.smk"
+    configfile:
+        "../workflows/reference_files/2.4/config/default.yaml"
+
+
+##### CONFIGURATION FILES #####
+
+
+
+# Load module-specific configuration
+
+configfile: "../modules/slms_3/1.0/config/default.yaml"
+configfile: "../modules/sage/1.0/config/default.yaml"
+configfile: "../modules/manta/2.3/config/default.yaml"
+configfile: "../modules/gridss/1.1/config/default.yaml"
+configfile: "../modules/strelka/1.1/config/default.yaml"
+configfile: "../modules/lofreq/1.0/config/default.yaml"
+configfile: "../modules/starfish/2.0/config/default.yaml"
+configfile: "../modules/controlfreec/1.1/config/default.yaml"
+configfile: "../modules/utils/2.1/config/default.yaml"
+configfile: "../modules/picard_qc/1.0/config/default.yaml"
+configfile: "../modules/bam2fastq/1.2/config/default.yaml"
+#configfile: "../modules/vcf2maf/1.2/config/default.yaml"
```

Comment:
If the use of vcf2maf should be optional you need to make that its own case. Make it an easy option for the user (for example a flag, parameter, or single variable toggle) rather than making the user dig through all of the code to uncomment multiple lines of code in different files every time they want to use it or stop using it.
A few files where I found commented out vcf2maf (not comprehensive):
`demo/capture_Snakefile.smk` 
`demo/genome_Snakefile.smk`
`demo/mrna_Snakefile.smk`

Reply:
vcf2maf is a module under progress, that's why it is commented